package.json

```diff
@@ -117,18 +117,11 @@
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "lint": "yarn eslint ."
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
```

Comment:
why

Reply:
I commented. Besides,  there is no problem when production and dev have same config, I think.
![image](https://user-images.githubusercontent.com/33005392/185056150-37c20bef-5e21-4b95-b1a1-90805ab0db02.png)